exec/java-exec/src/main/java/org/apache/drill/exec/planner/physical/ScanPrel.java

```diff
@@ -151,12 +151,12 @@ public RelOptCost computeSelfCost(final RelOptPlanner planner, RelMetadataQuery
 
   @Override
   public SelectionVectorMode[] getSupportedEncodings() {
-    return SelectionVectorMode.DEFAULT;
+    return SelectionVectorMode.NONE_AND_TWO;
   }
 
   @Override
   public SelectionVectorMode getEncoding() {
-    return SelectionVectorMode.NONE;
+    return SelectionVectorMode.TWO_BYTE;
```

Comment:
What do you mean by `NONE_AND_TWO ` ,since the return is `SelectionVectorMode` not an array. If JPPD is not applicable, to `ScanBatch` ,the output container's build schema would always be `SelectionVectorMode.NONE` , and will not generate SV2.

Reply:
Makes sense.  I was thinking of the other API which returns an array.